pygmt/datasets/samples.py

```diff
@@ -30,6 +31,7 @@ def list_sample_data():
         "mars_shape": "Table of topographic signature of the hemispheric dichotomy of "
         " Mars from Smith and Zuber (1996)",
         "ocean_ridge_points": "Table of ocean ridge points for the entire world",
+        "static_earth_relief": "Sample grid used for testing",
```

Comment:
```suggestion
        "static_earth_relief": "Sample grid used for testing based on Sandwell et al., 2022",
```

Reply:
Do we actually want `list_sample_data()` to print this `static_earth_relief` file? I thought this was meant to be non-user-facing?

Reply:
I suggested adding it to both for consistency with the other functions for loading data from the cache, but I don't have a strong preference for it to be an user-facing feature. If we do have it as a strictly internal feature, would it be better in pygmt.helpers.testing rather than this module?

Reply:
I didn't even think about using it in helpers, but I think it makes more sense there. We don't intend for this grid to be used by any users.